src/main/java/io/codelex/Test/Exercise2/Apple.java

```diff
@@ -0,0 +1,23 @@
+package io.codelex.Test.Exercise2;
+
+public class Apple extends Basket{
```

Comment:
Kāpēc Apple ir Basket child klase? Vai tad ābols pēc būtības ir grozs? Uzdevumā bija ļoti konkrēti uzrakstīts:
`Create two empty classes Apple and Mushroom.` t.i. šim klasēm jābūt pilnibā tukšām tai skaitā bez extends